skiko/src/jvmMain/kotlin/org/jetbrains/skiko/redrawer/WindowsSoftwareRedrawer.kt

```diff
@@ -0,0 +1,63 @@
+package org.jetbrains.skiko.redrawer
```

Comment:
Looks very similar to Linux version, shalln't we commonize?

Reply:
Done.